File: 프로그래머스/0/120826. 특정 문자 제거하기/특정 문자 제거하기.swift
import Foundation

func solution(_ my_string:String, _ letter:String) -> String {
    // return my_string.filter { String($0) != letter }
    var result = ""
    
    for chr in my_string {
        
        if String(chr) != letter {
            result += String(chr)
        }
    }
    return result
}

File: 프로그래머스/0/120826. 특정 문자 제거하기/특정 문자 제거하기.js
// function solution(my_string, letter) {
//     return my_string.split('').filter(a => a !== letter).join('');
// }

// letter를 기준으로 쪼개져서 결국 letter 없어짐
function solution(my_string, letter) {
    return my_string.split(letter).join('');
}